susemanager-utils/susemanager-sls/src/modules/uyuni_config.py

```diff
@@ -1116,6 +1343,19 @@ def systemgroup_create(name, descr, org_admin_user=None, org_admin_password=None
     return UyuniSystemgroup(org_admin_user, org_admin_password).create(name=name, description=descr)
 
 
+# FIXME: Should we keep the fallback mechanism to the admin user in the authentication?
+#        That could be dangerous if we didn't pass any authentication on some methods
```

Comment:
Not sure I understand the doubt here, let's clarify this before merging.

Reply:
Doesn't make sense anymore, can be removed